core/controllers/base_test.py

```diff
@@ -329,7 +329,7 @@ def _extract_keys_from_json_file(self, filename):
         )).keys())
 
     def _extract_keys_from_html_file(self, filename):
-        regex_pattern = r'(I18N_[A-Z/_\d]*)'
+        regex_pattern = r'(I18N_[A-Z/_\d]*)(?=.*_I18N_IDS$).*$'
```

Comment:
I think this no longer captures the stuff it used to capture.

Why not just add a `\b` to the front of the original regex? Would that work?